test/fixture/config

```diff
@@ -16,4 +16,4 @@ Host *
 Host tahoe?
   User nil
   ForwardAgent true
-  ProxyCommand ssh -q gateway -W %h:%p
+  ProxyCommand "ssh -q gateway -W %h:%p"
```

Comment:
@dotnil I'd like to get your thoughts on this. It looks like spaces in `ProxyCommand` are allowed, but not in `IdentityFile`. I haven't dug into why this is, but it seems inconsistent. Right now, the code will parse values until the end of line and then indiscriminately wrap them with double quotes when stringified.

Reply:
`IdentityFile` has always been different. For other parameters, the first obtained value is used. For `IdentityFile`, and the rarely used `CertificateFile`, all matches values will be tried. That's why the value `IdentityFile` in the object returned by `SSHConfig#compute` is an Array instead of a String.

In this case, I'd go with treat `IdentityFile` differently, yet again. If the value of `IdentityFile` contains space while stringify, wrap it with double quotes. For other parameters, there's no need for this logic.

If `CertitficateFile` is already been used, the special treatment shall be applied also.